webapp/Connector/src/component/Started.js

```diff
@@ -22,52 +22,71 @@ Ext.define('Connector.component.Started', {
-                                '<table class="tile started-row-2" id="home-video">',
-                                '<tr><td><h3 class="tile-title">Get started!</h3></td></tr>',
-                                '<tr><td><div class="home_video backgroundimage"></div></td></tr>',
-                                '<tr><td>Watch the most powerful ways to explore the DataSpace.</td></tr>',
-                                '</table>',
-                            '</div></td>',
-                        '</tr></table>',
+                        '<div class="get-started get-started-multiRow">',
+                        '<table class="tile-group">',
+                        '<tr>',
+                            '<td class="tile tile-image"><div class="home_video backgroundimage"></div></td>',
+                            '<td class="tile tile-image"><div class="home_bar backgroundimage"></div></td>',
+                            '<td class="tile tile-image"><div class="home_try_it_out backgroundimage"></div></td>',
+                            '<td class="tile tile-image"><div class="home_integrated_data backgroundimage"></div></td>',
+                        '</tr>',
+                        '<tr>',
+                            '<td class="tile-detail" style="padding-left:5%;"><h3 class="tile-title">Watch the <a style="cursor:pointer" class="tile" id="home-video">Get Started</a> video</h3>',
+                                '<div><table>',
+                                    '<tr><td>See the most powerful ways to explore the DataSpace.</td></tr>',
+                                '</table></div>',
+                            '</td>',
+                            '<td class="tile-detail" style="padding-left:10%;">',
+                                '<div id="tours-wiki"><table>',
+                                    '<tr><td>{[this.getTakeATourWiki()]}</td></tr>',
+                                '</table></div>',
+                            '</td>',
+                            '<td class="tile-detail" style="padding-left:10%"><h3 class="tile-title">Try it out</h3>',
+                                '<div><table>',
+                                    '<tr><td class="tile-detail-static"><a href=\'#learn/learn/Study/\' target="_blank">Learn about</a> studies, products, assays, antibodies, ad publications</td></td></tr>',
+                                    '<tr><td class="tile-detail-static"><a href=\'#summary\' target="_blank">Find subjects</a> with common characteristics</td></td></tr>',
+                                    '<tr><td class="tile-detail-static"><a href=\'#chart\' target="_blank">Plot</a> assay results across studies and years of research</td></td></tr>',
+                                    '<tr><td class="tile-detail-static">Compare <a href=\'#mabgrid\' target="_blank">Monoclonal antibodies</a> and their neutralization curves</td></td></tr>',
+                                '</table></div>',
```

Comment:
@labkey-bpatel 
Did the spec say to open these links in another tab (I didn't notice it)? Seems like we would just want to navigate within the app.

Reply:
Ah ok! wasn't too clear on the default behavior so went with opening in another tab. 
But now updated - clicking on those links will open the page in the same tab.